src/components/settings/Settings.h

```diff
@@ -172,6 +182,7 @@ namespace Pinetime {
         Notification notificationStatus = Notification::ON;
 
         uint8_t clockFace = 0;
+        uint8_t chimesState = 0;
```

Comment:
I think the code would be clearer if you used a `enum class` instead of a plain integer : instead of comparing chimeState with 1, 2 or 3, you would compare it with `ChimeOptions::None`, `ChimeOptions::Hours`, `ChimeOptions::HalfHours`, for example

Reply:
Perfect! Thank you very much!